### PR TITLE
feat: setup fung-sdk/checkout module

### PR DESCRIPTION
**Pull request by ashwini-fung**

### CODE REVIEW

File: checkout/src/checkout.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+export default class Checkout extends EventTarget {
+  protected checkoutId: string;
+
+  protected containerId: string;
+
+  protected sandbox: boolean;
+
+  constructor({
+    checkoutId,
```
Comment (niveksalim) [inline checkout/src/checkout.ts:15]: We might want to accept some options (stylings) as well?

Comment (ashwini-fung) [reply checkout/src/checkout.ts:15]: Yea, that's correct, but that's out of the scope of this ticket.

File: checkout/package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@fung-sdk/checkout",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "types": "./dist/types/index.d.ts",
+  "main": "./dist/index.js",
+  "scripts": {
```
[the remark this answers is not in the document]
Comment (ashwini-fung) [reply checkout/package.json]: that's a good one!

**Review by niveksalim — APPROVED** [review]

**Review by Yehonal — APPROVED** [review]
Looks good to me! GJ @ashwini-fung

File: packages/checkout/src/config.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export const CHECKOUT_ENDPOINT_SBX = 'https://sbx.fung.money';
```
Comment (cinjoff) [inline packages/checkout/src/config.ts:1]: Could we also have our DEV endpoint as an option? Merchants don't have API keys there so they could never use it

Comment (cinjoff) [reply packages/checkout/src/config.ts:1]: @ashwini-fung

**Pull request merged**